build_stats.sh

```diff
@@ -0,0 +1,34 @@
+#! /bin/sh
+
+INST_DIR=$HOME/R-inst
+mkdir -p ${INST_DIR}
+R_LIBS_USER=`pwd`/R
+mkdir -p ${R_LIBS_USER}
+
+# Download a recent version of R (must be 3.3.1 or later).
+wget https://cran.r-project.org/src/base/R-3/R-3.3.2.tar.gz
+tar -xvzf R-3.3.2.tar.gz
+cd R-3.3.2
+
+# Build R.
+./configure --prefix=${INST_DIR} --enable-R-shlib
+make
+make install
+cd ..
+
+# Correct PATH and LD_LIBRARY_PATH variables.
+export PATH=${INST_DIR}:${PATH}
+export LD_LIBRARY_PATH=${INST_DIR}/lib/R/lib:${LD_LIBRARY_PATH}
+
+echo "Please add ${INST_DIR}/bin to your PATH environment variable and \
```

Comment:
This works, but you need to move all of the messages to the end, as pip scrolls a load of stuff up the screen.